content/kubermatic/main/architecture/concept/kkp-concepts/networking/_index.en.md

```diff
@@ -14,16 +14,16 @@ The [expose strategy]({{< ref "../../../../tutorials-howtos/networking/expose-st
 This section explains how the connection between user clusters and the control plane is established, as well as the general networking concept in KKP.
 
 
-![KKP Network](@/images/concepts/architecture/network.png?classes=shadow,border "This diagram illustrated the necessary connections for KKP.")
+![KKP Network](images/network.png?classes=shadow,border "This diagram illustrated the necessary connections for KKP.")
```

Comment:
randomly: "illustrated"? So what does the diagram show now? Does it not illustrate the connections anymore? :grin: Maybe this could be reworded to "illustrates"

Reply:
fixed